### PR TITLE
ddl: Fix error format and clean up error trace.

### PR DESCRIPTION
**Pull request by zimulala**
Fix error format and clean up error trace.

### CODE REVIEW

**Comment by zimulala** [comment]
PTAL @hanfei1991 @coocood

File: ddl/schema.go [hunk 1]
```diff
@@ -81,7 +81,7 @@ func (d *ddl) onDropSchema(t *meta.Meta, job *model.Job) error {
 	}
 	if dbInfo == nil {
 		job.State = model.JobCancelled
-		return errors.Trace(infoschema.ErrDatabaseDropExists)
+		return infoschema.ErrDatabaseDropExists.Gen("Can't drop database; database doesn't exist")
```
Comment (hanfei1991) [inline ddl/schema.go]: It's better to create a error like "ErrDatabaseDropExistsWithoutColName", or use errors.New(). `Gen` is not recommanded.

Comment (coocood) [inline ddl/schema.go]: Or pass an empty string as argument.

**Comment by zimulala** [comment]
PTAL @hanfei1991 @coocood

**Comment by hanfei1991** [comment]
LGTM

**Pull request merged**